binderhub/main.py

```diff
@@ -7,6 +7,59 @@
 
 from .base import BaseHandler
 
+from tornado.escape import url_escape, url_unescape
+import json
+
+class BinderHandler(BaseHandler):
```

Comment:
This can just be inside BaseHandler no? Is there a reason this is its own class?

Reply:
No particular reason it was for base hander to not be too big. Moved there.